website/source/docs/agent/config_entries.html.md

```diff
@@ -21,53 +22,13 @@ Kind = "<supported kind>"
 Name = "<name of entry>"
 ```
 
-The two supported `Kind` configuration entries are detailed below.
+The supported `Kind` names for configuration entries are:
 
-## Configuration Entry Kinds
```

Comment:
We might need to correct or redirect links from here to the new proxy and serviced default pages.

Reply:
Done (within this repo).